src/lotus/client.rs

```diff
@@ -34,6 +34,8 @@ use crate::lotus::message::CIDMap;
 use crate::lotus::{LotusClient, NetworkVersion};
 use crate::manager::SubnetInfo;
 
+pub type DefaultLotusJsonRPCClient = LotusJsonRPCClient<JsonRpcClientImpl>;
```

Comment:
Why are you setting this alias?

Reply:
It's in the checkpoint, we need to specify the client lotus type, with alias, it's just much easier, otherwise every time, we need to write `LotusJsonRPCClient<JsonRpcClientImpl>`, it's a lot verbose and not intuitive as compared to `DefaultLotusJsonRPCClient`